### PR TITLE
Rebase 4-10-okd-2022-04-23-131357

### PR DESCRIPTION
**Pull request by sallyom**
Related Issue: https://github.com/redhat-et/microshift/pull/674

### CODE REVIEW

File: packaging/images/openshift-ci/Dockerfile [hunk 1]
```diff
@@ -8,7 +8,14 @@ WORKDIR $GOPATH/src/github.com/redhat-et/microshift
 
 COPY . .
 
-RUN yum install gpgme-devel glibc-static libassuan-devel -y
+RUN yum install gpgme-devel glibc-static libassuan-devel wget -y
+
+# update to go-1.18
```
Comment (fzdarsky) [inline packaging/images/openshift-ci/Dockerfile:13]: Is go-1.18 the right version for 4.10? Looks like 4.11 is using 1.17 still.

Comment (sallyom) [reply packaging/images/openshift-ci/Dockerfile:13]: great, yea, probably so, will do

Comment (sallyom) [reply packaging/images/openshift-ci/Dockerfile]: updated!

**Comment by sallyom** [comment]
CI is still using `openshift/release:rhel-8-release-golang-1.16-openshift-4.10 into pipeline:root` but golang-1.17 is required :shrug:

**Comment by mangelajo** [comment]
I tagged this container with the new version, since it won't really change:

https://quay.io/repository/microshift/flannel-cni?tab=tags

**Comment by mangelajo** [comment]
@sallyom all tests passed :D :-D congrats

**Comment by mangelajo** [comment]
/lgtm sally, excellent work as Ricardo said, thanks a lot for the help here!!

I have also added [this](https://github.com/redhat-et/microshift/pull/683) to build the community images, we can merge it after yours.

**Pull request merged**